extension/sigv4authextension/signingroundtripper_test.go

```diff
@@ -152,6 +158,20 @@ func TestInferServiceAndRegion(t *testing.T) {
 			"service",
 			"region",
 		},
+		{
+			"match_with_config",
```

Comment:
These tests should have unique names that describe what they're testing.  It's not clear what they're doing.  I suspect you want tests more like the `amp_service_and_region_match_with_no_config` case above, but with the respective services.

Reply:
Good call, updated.